README.md

```diff
@@ -17,3 +17,36 @@ Setup a systemd timer that cleans up unused and dangling images every night by r
 ### docker.compose
 
 Install `docker-compose` package.
+
+### docker.pillar-users
+
+Populate the `docker` group with a list of usernames extracted from Salt pillar data.
+The pillar of the targeted minion should contain the following, where each subkey (`FIRSTSET`, `SECONDSET`, ...) and associated list of users may originate from a different pillar file targeted to the minion (e.g. due to overlapping wildcard matches in pillar targeting).
+
+```
+docker-users:
+    ----------
+    FIRSTSET:
+        - user_A
+        - user_B
+    SECONDSET:
+        - user_A
+        - user_C
+```
+
+The resulting list of users in the dockergroup for this example would be `user_A, user_B, user_C`.
+The given exampel pillar data would be the result of targeting both the following onto some minion:
+
+```
+docker-users:
+    FIRSTSET:
+        - user_A
+        - user_B
+```
+
+```
+docker-users:
+    SECONDSET:
+        - user_A
+        - user_C
```

Comment:
I get the idea of multiple sets, but (like before) I think it would be cleaner and more simple to use a simple array, and then use Saltstack selectors in the `top.sls` to select the appropriate sets for the machines (or, if that's not fine grained enough, an `{% if %} {% else %}` Jinja statement. (But this is a topic for the upcoming meeting anyway).